data/xml/1995.iwpt.xml

```diff
@@ -0,0 +1,287 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="1995.iwpt">
+  <volume id="1">
+    <meta>
+      <booktitle>Fourth International Workshop on Parsing Technologies</booktitle>
+      <publisher>ACL</publisher>
+      <address>Prague and Karlovy Vary, Czech Republic</address>
+      <month>September 20-24</month>
```

Comment:
Looks like an editor got mis-parsed.

Reply:
no, it's a town ("Charles's Baths").  they had a tradition of having the second part of the conference in a resort/tourist area.